lib/inspector/inspector_text_styles.dart

```diff
@@ -0,0 +1,13 @@
+import '../ui/fake_flutter/fake_flutter.dart';
+
+final TextStyle grayed = TextStyle(color: Colors.grey.shade500);
+const TextStyle regular = TextStyle(color: Colors.black);
+final TextStyle warning = TextStyle(color: Colors.orange.shade500);
+final TextStyle error = TextStyle(color: Colors.red.shade500);
+final TextStyle link = TextStyle(
+    color: Colors.blue.shade700, decoration: TextDecoration.underline);
+
+const TextStyle regularBold =
+    TextStyle(color: Colors.black, fontWeight: FontWeight.w700);
+const TextStyle regularItalic =
+    TextStyle(color: Colors.black, fontStyle: FontStyle.italic);
```

Comment:
Will we have use for these styles outside of the inspector? If so, maybe consider moving them to ui_utils?

Reply:
these are generally inspector specific as most of them match inspector DiagnosticNode logging levels. That said if they are useful elsewhere in the future feel free to move out of the inspector directory.